### PR TITLE
[terra-slide-panel-manager] Updated docs to use MDX syntax

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Updated terra-slide panel-manager docs to use MDX syntax

### CODE REVIEW

**Comment by avinashg1994** [comment]
Can you interchange the order of the "changelog" and "upgrade guide" for consistency.
Since majority of the component's documentation have the "Upgrade guide" before the "changelog".

**Comment by saket2403** [comment]
> Can you interchange the order of the "changelog" and "upgrade guide" for consistency.
> Since majority of the component's documentation have the "Upgrade guide" before the "changelog".

Changed 76d1e22

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 1]
```diff
@@ -0,0 +1,306 @@
+/* eslint-disable max-classes-per-file */
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: Can you add some information on this new example?

Comment (saket2403) [reply packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: Added 35c8ed6

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+  </div>
+);
+
+SlidePanelManagerExample.propTypes = {
+  behavior: PropTypes.string,
+  showAccessory: PropTypes.bool,
+};
+
+const SquishSlideGroupManager = () => (
+  <SlidePanelManagerExample behavior="squish-2" />
```
Comment (mjhenkes) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: I don't believe 'squish-2' is a valid behaviour and this example is acting as overlay.

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerExample.jsx [hunk 3]
```diff
@@ -298,5 +298,9 @@ SlidePanelManagerExample.propTypes = {
   showAccessory: PropTypes.bool,
 };
 
-export default SlidePanelManagerExample;
+const OverlaySlideGroupManager = () => (
+  <SlidePanelManagerExample behavior="overlay-1" />
```
Comment (mjhenkes) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerExample.jsx]:
```suggestion
  <SlidePanelManagerExample />
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerAccessoryExample.jsx [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+  </div>
+);
+
+SlidePanelManagerExample.propTypes = {
+  behavior: PropTypes.string,
+  showAccessory: PropTypes.bool,
+};
+
+const AccessorySlideGroupManager = () => (
+  <SlidePanelManagerExample behavior="squish-3" showAccessory />
```
Comment (mjhenkes) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerAccessoryExample.jsx]:
```suggestion
  <SlidePanelManagerExample showAccessory />
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx (hunk 2 above)
Comment (saket2403) [reply packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: Changed ab9a6b8

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerExample.jsx (hunk 3 above)
Comment (saket2403) [reply packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerExample.jsx]: ab9a6b8

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerAccessoryExample.jsx (hunk 4 above)
Comment (saket2403) [reply packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerAccessoryExample.jsx]: Fixed ab9a6b8

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/slide-panel-manager/SlidePanelManager.1.doc.mdx [hunk 5]
```diff
@@ -45,6 +55,16 @@ The `disclosureAccessory` prop allows consumers of the SlidePanelManager to rend
...
+- [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+- [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+- [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+
+[View Component Source Code](https://github.com/cerner/terra-framework/tree/master/packages/terra-slide-panel-manager)
+
+## Examples 
+<SlidePanelManagerExample title='Overlay Example' />
+<SlidePanelManagerSquishExample title='Squish Example' />
+<SlidePanelManagerAccessoryExample  title='Disclosure Accessory Example' />
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/slide-panel-manager/SlidePanelManager.1.doc.mdx]:
```suggestion
<SlidePanelManagerExample title="Overlay Example" />
<SlidePanelManagerSquishExample title="Squish Example" />
<SlidePanelManagerAccessoryExample  title="Disclosure Accessory Example" />
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+  }
+}
+
+ContentComponent.contextType = DisclosureManagerContext;
+ContentComponent.propTypes = {
+  disclosureType: PropTypes.string,
+  panelBehavior: PropTypes.oneOf(['overlay', 'squish']),
+};
+
+const SlidePanelManagerExample = ({ behavior, showAccessory }) => (
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: Lets remove the props from this example & write specifically for the Squish example

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+  <div className={cx('example-wrapper')}>
+    <SlidePanelManager
+      panelBehavior={behavior}
+      disclosureAccessory={showAccessory ? (
+        <div className={cx('disclosure-accessory')}>
+          Disclosure Accessory
+        </div>
+      ) : undefined}
+    >
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]:
```suggestion
    <SlidePanelManager panelBehavior="squish">
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+  </div>
+);
+
+SlidePanelManagerExample.propTypes = {
+  behavior: PropTypes.string,
+  showAccessory: PropTypes.bool,
+};
+
+const SquishSlideGroupManager = () => (
+  <SlidePanelManagerExample behavior="squish" />
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]:
```suggestion
  <SlidePanelManagerExample />
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+);
+
+export default SquishSlideGroupManager;
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: Why not export the `SlidePanelManagerExample` directly so this is easier for consumers to look at?

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+const WIDTH_KEYS = Object.keys(availableDisclosureWidths);
+
+const generateDimensionOptions = values => (
+  values.map((currentValue, index) => {
+    const keyValue = index;
+    return <option key={keyValue} value={currentValue}>{currentValue}</option>;
+  })
+);
+
+class ContentComponent extends React.Component {
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]: Can we write this new example with hooks?

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+      </button>
+    );
+  }
+
+  renderForm() {
+    return (
+      <form>
+        <label htmlFor={this.getId(`disclosureHeight${this.props.panelBehavior}`)}>Pop Content Height</label>
+        <select id={this.getId(`disclosureHeight${this.props.panelBehavior}`)} name="disclosureHeight" value={this.state.disclosureHeight} onChange={this.handleSelectChange}>
+          {generateDimensionOptions(HEIGHT_KEYS)}
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]:
```suggestion
        <label htmlFor={this.getId(`disclosureHeightSquish`)}>Pop Content Height</label>
        <select id={this.getId(`disclosureHeightSquish}`)} name="disclosureHeight" value={this.state.disclosureHeight} onChange={this.handleSelectChange}>
          {generateDimensionOptions(HEIGHT_KEYS)}
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 12]
```diff
@@ -0,0 +1,307 @@
...
+        </select>
+        <br />
+        <br />
+        <label htmlFor={this.getId(`disclosureWidth${this.props.panelBehavior}`)}>Pop Content Width</label>
+        <select id={this.getId(`disclosureWidth${this.props.panelBehavior}`)} name="disclosureWidth" value={this.state.disclosureWidth} onChange={this.handleSelectChange}>
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]:
```suggestion
        <label htmlFor={this.getId(`disclosureWidthSquish`)}>Pop Content Width</label>
        <select id={this.getId(`disclosureWidthSquish`)} name="disclosureWidth" value={this.state.disclosureWidth} onChange={this.handleSelectChange}>
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 13]
```diff
@@ -0,0 +1,307 @@
...
+  <div className={cx('example-wrapper')}>
+    <SlidePanelManager
+      panelBehavior={behavior}
+      disclosureAccessory={showAccessory ? (
+        <div className={cx('disclosure-accessory')}>
+          Disclosure Accessory
+        </div>
+      ) : undefined}
+    >
+      <ContentComponent panelBehavior={behavior} disclosureType="panel" />
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]:
```suggestion
      <ContentComponent />
```

File: packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx [hunk 14]
```diff
@@ -0,0 +1,307 @@
...
+    return (
+      <button
+        type="button"
+        onClick={() => {
+          disclosureManager.disclose({
+            preferredType: disclosureType,
+            dimensions: { height: this.state.disclosureHeight, width: this.state.disclosureWidth },
+            content: {
+              key: 'Content-Disclosure-Dimensions',
+              component: <DisclosureComponent name="Disclosure Component" disclosureType={disclosureType} />,
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx]:
```suggestion
              component: <DisclosureComponent name="Disclosure Component" disclosureType="panel" />,
```

**Comment by emilyrohrbough** [comment]
@saket2403 any update on this?

**Pull request merged**